### DISCUSSION TITLE
How to disable statusline but keep the shortcut tip area?

### DISCUSSION THREAD

**Discussion by ravenxrz** [zellij-org/zellij#1270] [Q&A]
Like the screenshot below.

![image](https://user-images.githubusercontent.com/44746032/160092417-b359dd3c-e815-4771-aeb8-6c7adb5dbedd.png)

The shortcuts prompt is redundant when users get accustomed to it. But I do need the tip area to show status like pane entering FULLSCREEN.

**Comment by jaeheonji** [comment]
Hi! Unfortunately, This is currently not possible. It is expected that it will be possible if plugin configuration feature is implemented in the future.

**Comment by Mehdi-YC** [comment]
Hello , please is this still not possible ? 
also can you provide some template for minimal , tmux like setup please ?

**Comment by aljazerzen** [comment]
These plugins are used in your layout, so they are loaded automatically.

To remove the status bar, you need to change the layout.

To do that:

- dump the default layout:
  ```
  mkdir ~/.config/zellij/layouts
  zellij setup --dump-layout default > ~/.config/zellij/layouts/my_layout.kdl
  ```

- edit `~/.config/zellij/layouts/my_layout.kdl` and remove the status bar at the bottom,

- edit `~/.config/zellij/config.kdl` to set `default_layout "my_layout"`

That will fully remove both bars, since they are the same "status-bar" plugin. Personally, I like the following layout:

```kdl
layout {
    pane size=1 borderless=true {
        plugin location="compact-bar"
    }
    pane
}
```

**Comment by 23b00t** [comment]
I was looking around how to get everything I need, as I'm new to zellij still keybinding hints and a tab bar, in one bar. After getting here I found: [zellij faq](https://zellij.dev/documentation/faq.html?highlight=compact#the-ui-takes-up-too-much-space-what-can-i-do-about-it) 
Exactly doing what I was looking for. So as it quit well fits the title of this thread, although it doesn't seem to answer the OP, I thought I mention it. 

It's easy to include in custom layouts too, like:
```kdl
layout {
    default_tab_template name="main" {
        // compact-bar at the top
        pane size=1 borderless=true {
            plugin location="zellij:compact-bar" {
                tooltip "F1"
            }
        }
        children
    }

    tab name="nixos" cwd="~/nixos-config/" {
        pane
    }

  // other stuff
}
